src/Common/src/System/Drawing/KnownColor.cs

```diff
@@ -7,7 +7,11 @@
 namespace System.Drawing
 {
     [SuppressMessage("Microsoft.Design", "CA1008:EnumsShouldHaveZeroValue")]
+#if NETCOREAPP20
```

Comment:
By convention you should make the define `netcoreapp20` lowercase to match the configuration.

Reply:
Not that is a big deal but I would probably just #ifdef the visibility and not the type name.

Reply:
K, fixing now.